whatsagent/views.py

```diff
@@ -0,0 +1,139 @@
+import typing
+import json
```

Comment:
All signup related views should be moved to `/whatsagent/whatsagent/api/views.py`